javascript/react/intro-react/why-use-react.md

```diff
@@ -18,24 +18,22 @@ aspects:
   - introduction
 
 ---
-# Why use **React**?
+# Why use React?
 
 ---
 ## Content
 
-**React** is considered a great choice for building the **UI** of web applications - be it simple *SPAs*[1] or
-large projects with numerous authors due to the **self-containing** nature of its *components*.
+**React** is considered a great choice for building the **UI** of web applications - be it simple *SPAs*[1] or large projects with numerous authors due to the **self-containing** nature of its *components*.
 
 In addition to the previously stated reasons developers are eager to adopt this library, **React** comes with many more interesting features such as:
 
 - great *performance* as a result of the **Virtual DOM** diff algorithm
```

Comment:
Add footnote?

Reply:
Added the following footnote

The Virtual DOM (or VDOM) is nothing more but a programming concept in which a "virtual" representation of the UI is kept in memory. It is synced with the "real" DOM by using a library, which in our case is the `ReactDOM` library. The Virtual DOM provides increased performance because only the parts of the app that have been changed are re-rendered, instead of re-rendering the whole application.
```jsx
<div className="first" />

<div className="second"/>
```
In this case, React know to only modify the `className` on the underlying DOM node. For more information on the **diff algorithm** that React uses make sure to check out [this article](https://reactjs.org/docs/reconciliation.html).